src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -575,8 +608,13 @@ public void AnalyzeCommand_CachesResultsWhenComputingTargetHashesAndPersistingTo
             RunTests(persistLogFileToDisk: true);
         }
 
-        private static void RunTests(bool persistLogFileToDisk)
+        private static void RunTests(bool persistLogFileToDisk, bool raiseNotificationPerBinary = false, int expectedReturnCode = 0)
```

Comment:
>RunTests [](start = 28, length = 8)

This test is now officially a mess. Will refactor/clean up in a way that matches other high quality patterns LGolding has produced.